examples/manifest.yml

```diff
@@ -100,35 +100,41 @@ root:
         - name: arkitscenes
           python: python/arkitscenes
```

Comment:
This could use a better name imho

Reply:
Could be called "AR Scene Understanding"

Reply:
Or just "AR Scene"?

Reply:
"ARKit Scenes" it is (from #2420